lua/swenv/api.lua

```diff
@@ -97,6 +97,17 @@ local get_venvs_for = function(base_path, source, opts)
   return venvs
 end
 
+local get_pixi_base_path = function()
+  local current_dir = vim.fn.getcwd()
+  local pixi_root = Path:new(current_dir):joinpath('.pixi')
+
+  if not Path:new(pixi_root):exists() then
+    return nil
+  else
+    return Path:new(pixi_root) .. '/envs'
```

Comment:
since `pixi_root` is already a `Path` no need to create it again :)

```suggestion
  if not pixi_root:exists() then
    return nil
  else
    return pixi_root .. '/envs'
```